app/assets/stylesheets/application.scss

```diff
@@ -56,9 +63,55 @@ h6 {
   h3 {color: black;
     text-transform: uppercase;
     font-style: normal;
+    a {
```

Comment:
Rule declaration should be preceded by an empty line<br>Selector should have depth of applicability no greater than 2, but was 3